### PR TITLE
files: change the original_url logic 

### PR DESCRIPTION
**Pull request by monaawi**
* Populate original_url with url only if it is not already populated and if it url is not an s3 url and the url is not relative
* Populating the original_url is used in the afs cache in the migration

### CODE REVIEW

File: backend/inspirehep/records/api/literature.py [hunk 1]
```diff
@@ -327,7 +327,6 @@ def add_file(
             )
         return {
             "key": new_key,
-            "original_url": original_url or url,
```
Comment (michamos) [inline backend/inspirehep/records/api/literature.py]: so you never populate `original_url`? not sure I understand.

Comment (drjova) [inline backend/inspirehep/records/api/literature.py]: in workflows will work, what about migration? Is ``original_url`` populated correctly? cc @michamos

Comment (monaawi) [reply backend/inspirehep/records/api/literature.py]: no, we leave the ```original_url``` as it is. We don't change the ```original_url``` to be the ```url``` as this wrong - we would be storing afs and ```/api/files..``` in it

Comment (monaawi) [reply backend/inspirehep/records/api/literature.py]: yeah, I had also checked the workflows, we populate the ```original_url``` there. Not sure about the migration though

**Pull request merged**